src/docs/truffle/how-to/write-external-scripts.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Writing External Scripts / Command Plugins
+title: Write external scripts
 layout: docs.hbs
 ---
-# Writing External Scripts
+# Write External Scripts
```

Comment:
Casing discrepancy between left-hand nav and page title (and the metadata `<title>` too)